WordPress/src/main/java/org/wordpress/android/util/ContentDescriptionListAnnouncer.kt

```diff
@@ -0,0 +1,67 @@
+package org.wordpress.android.util
+
+import android.view.View
+import android.view.accessibility.AccessibilityEvent
+import android.view.accessibility.AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED
+import android.view.accessibility.AccessibilityNodeInfo.ACTION_CLICK
+import androidx.annotation.StringRes
+import androidx.core.view.AccessibilityDelegateCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+
+class ContentDescriptionListAnnouncer constructor(
+    @StringRes private val emptyListText: Int,
+    @StringRes private val endOfListText: Int,
+    @StringRes private val clickActionText: Int? = null,
+    private val contentDescriptions: List<String>,
+    targetView: View
+) {
+    private var currentIndex = 0
+
+    init {
```

Comment:
I am using the `init` to wire up the behavior. What I noticed is that the class is just instantiated and the functionality works so I am wondering if that's fine from a readable/architecture standpoint or should there be a function that's called so that the action being taken when the class is created is easily expressed.